config.php.default

```diff
@@ -49,3 +49,6 @@ $config['enable_billing'] = 1;
 
 # Enable the in-built services support (Nagios plugins)
 $config['show_services'] = 1;
+
+# Enable the MRTG style graphs if set to true
+$config['graph']['stacked'] = false;
```

Comment:
Maybe `$config['graphs']['stacked'] = false; `

We already have graphs for a few other config options.

Reply:
In fact, I'd say remove it from here anyway as we want users to use the webui to config this